terraform/README.md

```diff
@@ -11,3 +11,46 @@ The following resources must be created prior to running this Terraform:
  - a persistent storage account in which to store the state must exist
  - a sufficiently-permissioned role must exist and the credentials must be saved
    as secrets on this repository
+
+All subsequent steps are going to assume that these are both configured.
```

Comment:
What are you referring to when you say "both are configured"? Is it `$AZURE_STORAGE_ACCOUNT` and `$AZURE_STORAGE_RESOURCE_GROUP`

Reply:
The first bullet maps to what later is called `AZURE_STORAGE_ACCOUNT`, yeah. The role is a separate thing. But I wanted to not define these terms up front. This section will need to be significantly reworked soon when we move past a POC. I didn't want to tightly couple those terms too much.